linux_files/setup

```diff
@@ -271,7 +278,9 @@ then
 
     wget https://api.patrickwu.ml/wlinux/pulseaudio-for-wsl.zip
 
-    # implement a hard-coded sha check here for security
+    # implement a hard-coded sha check here for security, can you think of an elegant way to do so?
+
```

Comment:
`pulseaudio` for Windows hasn't been updated for ages, I believe it is stable enough to use hard-coded way to check SHA